_posts/2018-05-17-running-strimzi-on-openshift-online.md

```diff
@@ -0,0 +1,313 @@
+---
+layout: post
+title:  "Running Strimzi on OpenShift Online"
+date: 2018-05-17
+author: "Jakub Scholz"
+---
+
+OpenShift Online is a managed OpenShift as a Service offering from Red Hat. It has a Starter tier and Pro tier.
+The Starter tier is for free, but offers only very limited resources.
+In the Pro tier, you can buy additional resources.
+Can it be used to run Apache Kafka using the Strimzi project?
+Of course it can.
+But OpenShift Online has some limitations.
+So we will need to do some modifications to the default Strimzi installation files to get it up and running.
+
+<!--more-->
+
+Since OpenShift Online is a managed service, you will not get an account with cluster admin access.
+You will be only admin of the projects which you create.
+As such, you will be unable to create new roles.
+To follow the principle of least privilege, the default installation of Strimzi creates a new role which contains only the exact access rights which are needed by Strimzi.
+But OpenShift Online will not let you create such role.
+So instead, we need to reuse one of the existing roles.
+The cluster role named edit is suitable for this, since it contains all the privileges Strimzi needs to watch and manage resources.
```

Comment:
"The edit cluster role ..." ? In any case "edit" should be highlighted in some way (bold, apexes, ...)